docs/source/noise/index.rst

```diff
@@ -93,6 +93,104 @@ column shows the name of the noise type in the configuration system.
 Default Noise Types for Each Column
 -----------------------------------
 
+.. list-table:: Types of noise for each column
+  :widths: 20 20 20 20
+  :header-rows: 0
+
+  * - Column name
+    - Datasets present
+    - Types of noise
+    - Notes
+  * - First name
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Leave a field blank, use a fake name, make typos
+    -
+  * - Middle initial
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Leave a field blank, make typos
+    -
+  * - Last name
+    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
+    - Leave a field blank, use a fake name, make typos
+    - The list of fake names will be different than the first names
```

Comment:
```suggestion
    -
```

Ah sorry can we remove this too? I think it is confusing and already explained better in the details section.